### ISSUE TITLE
Three-finger workspace gesture hides HyperSpace due to Waybar activation

### ISSUE THREAD

**Issue by shadowelite-sec** [KZDKM/Hyprspace#9]
####  1. Three-finger workspace slide gesture activates Waybar and hides HyperSpace when toggle overview is enabled

https://github.com/KZDKM/Hyprspace/assets/54718375/1669db68-094f-4a0f-b930-2108e77856ee

####  2. I also noticed that when I launch HyperSpace, the current workspace reflects HyperSpace itself in the HyperSpace window, as shown in the above screenshot. Is that normal?

![image](https://github.com/KZDKM/Hyprspace/assets/54718375/f2e392c0-7db4-474d-8819-35d432a299ad)

here is my configuration , am i missing something ?
```
    overview {
    	workspaceBorderSize = 5
    	workspaceActiveBorder = $color3
	hideBackgroundLayers = 0
	centerAligned = 1
	showNewWorkspace = 1
	panelColor = rgba(00000000)
    }

```

**Comment by KZDKM** [comment]
I have not tested touchpad compatibility yet as the only laptop I have is a macbook with Asahi Linux installed and I would have to get a working function hook system for ARM to have this plugin on it. This should be a easy fix by re-hiding panels on workspace drag end. I will look into it tomorrow as it is getting late here.

As per the second issue, it is working as intended, as the current workspace layout gets squished by the overview panel and would leave a large empty space, rendering the panel in the workspace view covers up that area.

**Comment by KZDKM** [comment]
This was probably caused by a race condition of hiding / showing panels. Should be fixed by https://github.com/KZDKM/Hyprspace/commit/9a274c078574c716039efaa4f2a67bb26f079ea7. 

I could not test touchpad workspace swipe as I do not have the hardware at the moment. Please verify whether if the fix is effective on your setup.

**Comment by shadowelite-sec** [comment]
> This was probably caused by a race condition of hiding / showing panels. Should be fixed by [9a274c0](https://github.com/KZDKM/Hyprspace/commit/9a274c078574c716039efaa4f2a67bb26f079ea7).
> 
> I could not test touchpad workspace swipe as I do not have the hardware at the moment. Please verify whether if the fix is effective on your setup.

Its working now :+1:  , Thanks :smile:
